dan_layer/storage/src/consensus_models/block.rs

```diff
@@ -231,6 +245,7 @@ impl Block {
             0,
             IndexMap::new(),
             None,
+            EpochTime::now().as_u64(),
```

Comment:
This will be a problem as hashes will not match. I think let's pass in the same timestamp as the parent block.

Reply:
done